com.unity.render-pipelines.core/Runtime/Common/CoreUnsafeUtils.cs

```diff
@@ -150,6 +151,233 @@ public static void CopyTo<T>(this T[] list, void* dest, int count)
                 UnsafeUtility.WriteArrayElement<T>(dest, i, list[i]);
         }
 
+        private static void CalculateRadixParams(int radixBits, out int bitStates)
```

Comment:
Since we are adding some Sorting functions to Core I think it would be good to have at least some very basic unit tests written for these.

Reply:
Sounds good.

Reply:
Unit tests done! I had no idea we had functional tests :) will use them more often now.